DummyAiHttpServer/README.md

```diff
@@ -0,0 +1,28 @@
+# Simple http server that listens on port 7861 and endpoint /sdapi/v1/txt2img
```

Comment:
I tested new `ya-runtime-ai` release using this sample server.
Readme update with description how to init Python env.